pages/footer/footer.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<md-card layout-fill class="footer-background-color oppia-foundation-conditional-vertical-padding">
+  <md-card-content layout="column" layout-align-sm="center" layout-gt-sm="row" layout-align-gt-sm="center">
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">The Oppia Foundation</h4>
+      <p class="footer"><a>About</a></p>
+      <p class="footer"><a>Partner With Us</a></p>
+      <p class="footer"><a>Get Involved</a></p>
+      <p class="footer"><a>Donate</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Oppia</h4>
+      <p class="footer"><a>About</a></p>
+      <p class="footer"><a>Get Started</a></p>
+      <p class="footer"><a>Browse Topics</a></p>
+      <p class="footer"><a>Contribute to Our Lessons</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Contact us</h4>
+      <p class="footer"><a>info@oppiafoundation.org</a></p>
+    </md-content>
+    <md-content layout="column" layout-align="center center" layout-align-gt-sm="start center" class="footer-background-color oppia-foundation-horizontal-padding">
+      <h4 class="oppia-foundation-uppercase-text oppia-foundation-white-text footer-title">Follow us</h4>
+      <md-content  layout="row" layout-align="center center" layout-gt-md="row" class="footer-background-color">
+        <md-button class="social-media-button google-plus-button" md-no-ink>
+          <md-icon md-svg-src="/static/icons/google-plus-g-brands.svg"
+                  aria-label="G+ icon"
```

Comment:
Here and below: typically we indent this by one more space to match the attr in the row above it.

Reply:
I was going by tab spaces and wasn't sure what do do here. Thanks for the pointer :)